Crash on 'Edit Message'

The whole client crashes if a message gets edited.

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x50 pc=0x1328dc4]
goroutine 133 [running]:
github.com/thorsager/t-slacker/runtime.(*AppRuntime).renderMessageEvent(0xc00014c280, 0xc00009c280, 0x9, 0xc0000e2000, 0x1d)
    /Users/mit/go/src/github.com/thorsager/t-slacker/runtime/messages.go:30 +0x104
github.com/thorsager/t-slacker/runtime.(*AppRuntime).rtmEvent(0xc00014c280, 0xc0000ac6c0, 0xc00000e140)
    /Users/mit/go/src/github.com/thorsager/t-slacker/runtime/app_context.go:159 +0x39e
github.com/thorsager/t-slacker/connection.(*Connection).delegateEvents(0xc0000ac6c0)
    /Users/mit/go/src/github.com/thorsager/t-slacker/connection/slack.go:193 +0x3a
created by github.com/thorsager/t-slacker/connection.New
    /Users/mit/go/src/github.com/thorsager/t-slacker/connection/slack.go:184 +0x541
```

--- a/runtime/messages.go
+++ b/runtime/messages.go
@@ -18,11 +18,7 @@ var (
 
 func (c *AppRuntime) renderMessageEvent(msg *slack.MessageEvent) []byte {
 	switch msg.SubType {
-	case "message_replied":
-		return []byte("\nTODO: handle message-replies")
-	case "message_deleted":
-		return []byte("\n[#333333]a message was deleted.[-]")
-	default:
+	case "":
 		team := c.GetTeam(msg.Team)
 		t := asTime(msg.Timestamp)
 		uname := msg.User
@@ -37,6 +33,8 @@ func (c *AppRuntime) renderMessageEvent(msg *slack.MessageEvent) []byte {
 		parsedText += renderFiles(msg.Files)
 		return []byte(fmt.Sprintf("\n%.2d:%.2d [#666666]<[-][%s]%s[-][#666666]>[-] %s", t.Hour(), t.Minute(),
 			color, uname, parsedText))
+	default:
+		return []byte(fmt.Sprintf("\n[#333333]Event subType '%s' not yet handled (hidden=%t).[-]", msg.SubType, msg.Hidden))
 	}
 }
 

